api/migrations/extract-person-contacts.js

```diff
@@ -208,10 +208,22 @@ var createPerson = function(id, callback) {
  */
 var updateParents = function(id, callback) {
   var checkParent = function(facility, callback) {
-    if (!facility.parent || (Object.keys(facility.parent).length === 0)) {
+    if (!facility.parent) {
       return callback({ skip: true });
     }
 
+    // There have been cases of weird {} parents. Remove these, then skip
+    if (Object.keys(facility.parent).length === 0) {
+      delete facility.parent;
+      return db.medic.insert(facility, err => {
+        if (err) {
+          callback(err);
+        } else {
+          callback({ skip: true });
```

Comment:
I know it's not your code but as far as I can tell `callback({skip: true})` does the same thing as `callback()`. Consider removing these at the same time?

Reply:
It is weird, but it is different. This function is called via `async.waterfall`, and `{skip: true}` is treated by async as an error (because it's not `null, {skip: true}`), which means it skips the rest of the functions in the waterfall.